pom.xml

```diff
@@ -7,7 +7,7 @@
     </parent>
     <artifactId>clover</artifactId>
     <packaging>hpi</packaging>
-    <version>4.12.2-SNAPSHOT</version>
+    <version>${revision}${changelist}</version>
```

Comment:
How does this affect `mvn release:prepare release:perform`? It won't be able to update version number automatically, I guess.

Reply:
The version number is incremented automatically based on that syntax in many plugins, including the git plugin, and the git client plugin.